servus/serializable.cpp

```diff
@@ -23,37 +23,128 @@
 
 namespace servus
 {
+
+class Serializable::Impl
+{
+public:
+    void registerChangedCallback( const Serializable::ChangeFunc& func )
+    {
+        changed = func;
+    }
+
+    void registerDeserializedCallback(
+                                     const Serializable::DeserializeFunc& func )
+    {
+        deserialized = func;
+    }
+
+    void registerSerializeCallback( const Serializable::SerializeFunc& func )
+    {
+        serialize = func;
+    }
```

Comment:
No, a least for me it was just a code style comment :-)
instead of having these functions just _impl->serialize = func instead of _impl->registerChangedCallback( func )